book/src/design/commitment-tree.md

```diff
@@ -36,12 +36,17 @@ sage: EllipticCurve(GF(p), [0, 5]).count_points() == q
 True
 sage: EllipticCurve(GF(q), [0, 5]).count_points() == p
 True
-sage: Mod(5, p).is_square()
+sage: Mod(13, p).is_square()
 False
-sage: Mod(5, q).is_square()
+sage: Mod(13, q).is_square()
 False
 ```
 
+> Note: There are also no Pallas points with $x$-coordinate $0$, but we map the identity to
+> $(0, 0)$ within the circuit. Although $\mathsf{SinsemillaCommit}$ cannot return the identity
+> (the incomplete addition would return $\perp$ instead), it would arguably be confusing to
```

Comment:
Why $\perp$ rather than $\bot$? (The latter is what the spec uses, and reflects the meaning of the symbol as denoting the bottom element of a lattice, rather than its shape.)